src/data/i18n/locales/currentlySupportedLangs.jsx

```diff
@@ -4,13 +4,10 @@ import esData from 'react-intl/locale-data/es';
 import frData from 'react-intl/locale-data/fr';
 import zhData from 'react-intl/locale-data/zh';
 
-// PUT THESE BACK ONCE THEY EXIST AGAIN
-/*
```

Comment:
This change will make it so that our new message files are included in built output again, restoring master to an "able to be published" state once more.